missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/Game.java

```diff
@@ -416,11 +415,6 @@ private void checkTeamSize(Team team) {
     }
 
     public void resetGame() {
-        if (state == GameState.INGAME) {
-            stopGame();
-            return;
-        }
-
```

Comment:
`stopGame()` is the process to change the `GameState` from "INGAME" to "END". It's not usefull, if we would like to stop the game process now, because the END-process has a timer too.